app/lib/contentdm/importer.rb

```diff
@@ -43,17 +52,30 @@ def process_record(record)
       work.contributor = cdm_record.contributors
       work.subject = cdm_record.subjects
       work.description = cdm_record.descriptions
-      work.visibility = 'open'
+      work.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC
       save_work(cdm_record, work)
+      @collection.add_members(work.id)
       work
     end
 
+    ##
+    # @return [Array<String>] this returns the name of the collection based on the XML
+    def collection_name
+      [@doc.xpath("//collection_name").text]
+    end
+
+    ##
+    # @return [ActiveFedora::Base] return the collection object
+    def collection
+      Collection.new(collection_name).find_or_create
```

Comment:
Since we have 2 different classes named 'Collection', could you add the namespace here so that it will be clear that it's `::Contentdm::Collection` and not just `::Collection`